### ISSUE TITLE
Create add shelf modal

### ISSUE THREAD

**Issue by knjk04** [Project-Books/book-project#665]
**Describe the solution you'd like**

![image](https://user-images.githubusercontent.com/11173328/112597347-eb157f00-8e04-11eb-84d1-115564317e97.png)

Create an add shelf modal that is centered in the middle of the screen. It should also be responsive.

The modal should be brought up when the user clicks on 'Add shelf'. When the user clicks on the 'cancel' button in the modal, it should close.

**Comment by knjk04** [comment]
Blocked by #664

**Comment by olusolaa** [comment]
@knjk04 I can take this

**Comment by knjk04** [comment]
@olusolaa Sure, thanks!

**Comment by olusolaa** [comment]
Hi @knjk04 where can I find the React application for this project?

**Comment by olusolaa** [comment]
@knjk04 I can't access the MY_BOOKS route. 
I have started both frontend and backend locally and I tried signing up however there was no redirection. Is there something I'm missing?

**Comment by olusolaa** [comment]
Oh! thanks

**Comment by olusolaa** [comment]
@knjk04 My sincere apology for the delay and late response will revert in a few hours time.

**Comment by olusolaa** [comment]
I think I mistakenly committed node modules and the demo alongside. Should I send a new pull request?

**Comment by RivaD2** [comment]
@olusolaa I reviewed your PR and added in some comments for changes to be made. Nice work so far, and it is almost ready!
